modules/ROOT/pages/fd-project-history.adoc

```diff
@@ -9,24 +9,18 @@ endif::[]
 You can see a history of changes you make to a project in Flow Designer.
 The saved states are listed by the ID of the user who made the change, time, date, and activity. The activity name is based on the action you performed at save time.
 
-Being able to view a saved state of a project is useful if you want to:
-
-* Undo an inadvertent change you made to a project by reverting back to a working state.
-* Review changes made by other users to the project.
-* Retrieve older values of some project configurations.
+Being able to restore a saved state of a project is useful if you want to:
 
+* Undo an inadvertent change you made to a project and revert to the working state.
```

Comment:
Why did you change "a working state" back to "the working state"? There might be a number of saved states that were working.

Reply:
Not sure why this was changed back (as you say) or missed.